_config.yml

```diff
@@ -268,6 +268,12 @@ authors:
   daniel:
     name: Daniel Levenson
     github: dleve123
+  
+  anna:
+    name: Anna Carey
+    github: annacarey
+    twitter: anna_carey
+    site: https://annajcarey.com
```

Comment:
I get an SSL warning at this address but https://www.annajcarey.com seems to work fine!

http://annajcarey.com also seems to redirect to https://www.annajcarey.com.

Reply:
OMG i know i have heroku problems with my website. Maybe i should change it to `www.` in here

Reply:
Might be a good idea for now! You can always change/redirect later.

Also happy to pair on the DNS/SSL config sometime!

Reply:
YES i will def take you up on that